home-service/src/pages/BusinessPage/BusinessPage.tsx

```diff
@@ -0,0 +1,40 @@
+import styles from './BusinessPage.module.css';
+import { useContext, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import Button from '@/components/common/Button/Button';
+import { GiNotebook } from 'react-icons/gi';
+import BusinessInfoSection from '@/pages/SingleServicePage/BusinessInfoSection';
+import SimilarBusiness from '@/pages/SingleServicePage/SimilarBusiness';
+import { UserContext } from '@/context/UserContext';
+import { useBusiness } from '@/hooks/useBusinesses';
+
+const BusinessPage: React.FC = () => {
+  const { id } = useParams<{ id: string }>();
+  const [bookOpen, setBookOpen] = useState<boolean>(false);
+  const { isLoggedIn } = useContext(UserContext);
+  const { data: business, isLoading, error } = useBusiness();
+
+  const handleOpenModal = () => setBookOpen(true);
+  const handleCloseModal = () => setBookOpen(false);
+
+  if (!id || isLoading || error) return null;
```

Comment:
if (!business || isLoading || error) {
    return <div>There is no information</div>;
  }